packages/components/src/components/themed-scrollbars/themed-scrollbars.scss

```diff
@@ -48,8 +48,10 @@
             vertical-align: middle;
             width: unset;
             height: 40px;
+        }
 
-            @include mobile {
+        @include mobile {
+            & .dc-tabs__p2p-item--small {
```

Comment:
ditto here, components are meant to be platform / package agnostic, so any package specific (in this case P2P) overrides should be handled outside of the component